Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -0,0 +1,1384 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+
+! This file was ported from Lean 3 source module set_theory.ordinal.notation
+! leanprover-community/mathlib commit b67044ba53af18680e1dd246861d9584e968495d
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Init.Data.Ordering.Lemmas
+import Mathlib.SetTheory.Ordinal.Principal
+
+/-!
+# Ordinal notation
+
+Constructive ordinal arithmetic for ordinals below `ε₀`.
+
+We define a type `ONote`, with constructors `0 : ONote` and `ONote.oadd e n a` representing
+`ω ^ e * n + a`.
+We say that `o` is in Cantor normal form - `ONote.NF o` - if either `o = 0` or
+`o = ω ^ e * n + a` with `a < ω ^ e` and `a` in Cantor normal form.
+
+The type `NONote` is the type of ordinals below `ε₀` in Cantor normal form.
+Various operations (addition, subtraction, multiplication, power function)
+are defined on `ONote` and `NONote`.
+-/
+
+set_option linter.uppercaseLean3 false
+
+
+open Ordinal Order
+
+open Ordinal
+
+-- Porting note: the generated theorem gets lint.
+set_option genSizeOfSpec false in
+-- get notation for `ω`
```

Comment:
```suggestion
open Ordinal Order

-- Porting note: the generated theorem gets lint.
set_option genSizeOfSpec false in
```

The mathlib3 source had
```
open ordinal order
open_locale ordinal -- get notation for `ω`
```
that's where the comment  comes from

Reply:
It seems the comment has now vanished from the source. Could it be restored, please?

Reply:
Even without the notation for `ω`, `open Ordinal` is required.
And, in Lean 4, when an namespace is opened, its locale is automatically opened too, so `open_locale ordinal` and the comment is not needed anymore.

Reply:
Thanks, sorry to be slow.